x-pack/winlogbeat/module/sysmon/config/winlogbeat-sysmon.js

```diff
@@ -360,27 +371,62 @@ var sysmon = (function () {
         evt.Delete("winlog.event_data.DestinationIsIpv6");
     };
 
-    var addHashes = function (evt, hashField) {
-        var hashes = evt.Get(hashField);
+    var setRelatedIP = function (evt) {
+        var ips = [];
+        var sourceIP = evt.Get("source.ip");
+        if (sourceIP) {
+            ips.push(sourceIP);
```

Comment:
You can use evt.AppendTo("related.ip", sourceIP) so you don't have to manage the array yourself.

Bottom of this page: https://www.elastic.co/guide/en/beats/filebeat/current/processor-script.html